ResumeAR/CloudantRESTCall.swift

```diff
@@ -13,7 +13,7 @@ import SwiftyJSON
 class CloudantRESTCall {
 
     func getResumeInfo(classificationId: String, completionHandler: @escaping (_ result: JSON) -> Void){
-        let urlString = Credentials.CLOUDANT_URL.addingPercentEncoding(withAllowedCharacters: .urlFragmentAllowed)!
+        let urlString = Credentials.CLOUDANT_GET.addingPercentEncoding(withAllowedCharacters: .urlFragmentAllowed)!
```

Comment:
i like the one env var, so the GET call for a create would just be adding a `_find` at the end, after the encoding